packages/connectors/connector-xiaomi/README.md

```diff
@@ -0,0 +1,61 @@
+# Xiaomi connector
```

Comment:
Please rebase your branch to make sure the Xiaomi connector commit is not included in this PR.